src/schema/submission.yaml

```diff
@@ -308,6 +311,15 @@ classes:
       cell_barcode_offset:
         required: false
         recommended: true
+      cdna_read:
```

Comment:
It would be good to separate the slot additions related to single-cell data to another PR?